assets.go

```diff
@@ -0,0 +1,6 @@
+package tankism
```

Comment:
golang embed does not allow to import from parent (e.g ..) therefore we now have a asset graveyard file...